pkgs/top-level/all-packages.nix

```diff
@@ -2670,6 +2670,8 @@ with pkgs;
 
   emulationstation = callPackage ../applications/emulators/emulationstation { };
 
+  emulationstation-de = callPackage ../applications/emulators/emulationstation-de { };
+
```

Comment:
```suggestion
```
New top-level packages should go to `pkgs/by-name`  https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

Reply:
Thank you for the link, not sure how I missed that!